tool/generator/lib/src/projection/type.dart

```diff
@@ -251,6 +251,10 @@ class TypeProjection {
           'Pointer<${refTuple.nativeType}>', 'Pointer<${refTuple.dartType}>');
     }
 
+    if (typeIdentifier.typeArg?.baseType == BaseType.uint32Type) {
+      return const TypeTuple('Pointer<Uint32>', 'Pointer<Uint32>');
```

Comment:
This handles parameters like [IndexOf(T, UInt32)](https://docs.microsoft.com/en-us/uwp/api/windows.foundation.collections.ivectorview-1.indexof?view=winrt-22621#windows-foundation-collections-ivectorview-1-indexof(-0-system-uint32@))'s index:
```cs
  [MethodImpl(MethodImplOptions.InternalCall)]
  bool IndexOf([In] T value, out uint index);
```

We project these manually but there may be other methods that have "out uint32" parameters that we don't know about.

Reply:
Thanks. I've added a comment to that effect.